data_model/src/query/mod.rs

```diff
@@ -1308,6 +1344,9 @@ pub mod http {
             /// Payload
             pub payload: QueryPayload,
         }
+
+        /// End type of a query http clients can send to an endpoint.
+        pub type ClientQueryRequest = QueryRequest<SignedQuery>;
```

Comment:
I don't think there is any need to keep this in data_model?

Reply:
it can reside in `iroha_core` and be imported by cli

Reply:
It's used by `iroha_client` as well